05-Data_Ethics.Rmd

```diff
@@ -159,39 +159,76 @@ Possible guidelines could include (based on @garrison_genomic_2019, @garrison_ge
 
 > encourage researchers to consider cultural perspectives and traditional knowledge that has the potential to create stronger conclusions and better-informed, ethical, and respectful science.
 
 ## General Ethics Code
 
-maybe go into this: https://educationprojects.co.uk/medical-ethics-ethical-dilemmas-in-healthcare/
+Healthcare ethics, and by extension medical research ethics can be described in four pillars (this discussion from @melvin_medical_2020):
+
+### 1. The pillar of beneficence
+
+This pillar means that healthcare and its research must support the well-being of the patients. This includes reducing pain and helping to increase their overall quality of life.
+
+### 2. The pillar of non-maleficence
+
+Harm to patients must be minimized. This includes not causing intentional harm but also minimizing incompetency and accidental harm.
+This means for research studies and treatments, only conducting those where benefits outweigh the risks.
+
+### 3. The pillar of justice
+
+Healthcare must be provided to all equally and equitably regardless of race, gender, sex, socioeconomic class, religion, ethnicity, sexual orientation, age, or any other aspect of an individual's identity.
+Equity specifically means that those who are societally disadvantaged should be even further aided.
+
+### 4. The pillar of autonomy (self-determination)
+
+Patients need to make their own decisions about their health. They should be equipped with all information about their health but then respected and supported in their decision.
+This is otherwise includes informed consent [@commissioner_guide_2020], which means before a patient can truly consent they need to be fully informed of the risks, ins and outs of any procedure, treatment, or research study participation.
+
+## Code of Medical Ethics (1957) - Legal Binding of Patient-Physician Confidentiality
+
+1996 Health Insurance Portability and Accountability Act (HIPAA) protected health information and identifiable health information must not be shared with anyone outside of certain covered entities without consent. Covered entities include: clinicians, insurance companies, and health care government agencies.
 
-and this amazing timeline:
-https://www.usuhs.edu/sites/default/files/2021-06/TimelineofLandmarkEventsinBioethics.pdf
+## Genetic Information Nondiscrimination Act of 2008 (GINA)
 
-- informed consent etc. [@commissioner_guide_2020]
+The Genetic Information Nondiscrimination Act prohibits employers and health insurance companies from using genetic information to discriminate against individuals.
+This data would be highly dangerous and valuable if in the wrong hands. Health insurance companies are prohibited from obtaining genetic test data and using it to inform their premiums or other healthcare coverage related decisions.
 
-Code of Medical Ethics (1957) - Legal Binding of Patient-Physician Confidentiality 
```

Comment:
is this somewhere else now as a note? sorry it is hard to see cause somehow the spacing is different so almost everything looks changed :P

Reply:
Yeah I hate when it does that, I was about to start typing about this too which is why I deleted it, but I haven't yet added a description of this yet. I will add it back in for now

Reply:
I had kinda hoped to have a general historical context/timeline section here.

Reply:
I think there might be some other spots like that

Reply:
Oh this header is moved up: line 193